src/key.cpp

```diff
@@ -26,6 +26,17 @@
 
 #include "mococrw/openssl_wrap.h"
 
+namespace {
+void isEccCurveSupported(mococrw::openssl::ellipticCurveNid nid)
```

Comment:
Sort of a nitpick, feel free to ignore: Since this throws rather than returning a bool, I'd name this `ensureEccCurveSupported`.

Reply:
Done